.env.template

```diff
@@ -27,6 +27,7 @@ SENDGRID_TEMPLATE_DRIVE_SHARING_USER_REMOVED=d-0c9cbd0a649d4cee8eac6a4ae68736c9
 SENDGRID_TEMPLATE_DRIVE_SHARING_ROLE_UPDATED=d-0c9cbd0a649d4cee8eac6a4ae68736c9
 SENDGRID_TEMPLATE_SEND_LINK_CREATE_RECEIVER=d-eb5a1fd73d764e9991a25e2a0297f279
 SENDGRID_TEMPLATE_SEND_LINK_CREATE_SENDER=d-7889146930fa421083b4bf1cdcaedab3
+SENDGRID_TEMPLATE_DRIVE_UPDATE_USER_EMAIL=d-46a66194ad8e4bb6919d39e20d9c34d1
```

Comment:
please remember to add this variable @sg-gs

Reply:
Done